From efad0ea10d86a774d8e12c4c4b94504780e96642 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Wed, 14 Mar 2018 10:44:15 +0800
Subject: [PATCH 3482/5242] MLK-17794: ASoC: fsl: Add SND_SOC_IMX_CDNHDMI back

commit  3c770f14a8c6dfd38936f7cf8ca19470e1ca0aee from
https://source.codeaurora.org/external/imx/linux-imx.git

The FB_MX8_HDMI is removed, the dependency is changed
to DRM_IMX_HDP.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Reviewed-by: Sandor Yu <sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index df97a99..28f7475 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -517,7 +517,7 @@ config SND_SOC_IMX_AMIX
 
 config SND_SOC_IMX_CDNHDMI
 	tristate "SoC Audio support for i.MX boards with CDN HDMI port"
-	depends on FB_MX8_HDMI
+	depends on DRM_IMX_HDP
 	select SND_SOC_IMX_PCM_DMA
 	select SND_SOC_FSL_SAI
 	select SND_SOC_HDMI_CODEC
-- 
1.7.9.5

